deploy/values.yml

```diff
@@ -38,10 +38,10 @@ podAnnotations:
 
 resources:
   limits:
-    memory: 256Mi
+    memory: 96Mi
```

Comment:
This mem limit is way too low. It only give the app ~30Mi to work with before it runs out of mem and crashes with OOM. Setting the limit in the range of 256-512Mi should be fine as our nodes should be able to provide this. And for slightly bigger apps 1Gi is fine too.

Reply:
I was just trying to comply with what Nick and Sasha ended up agreeing to with regards to this (setting `limit` to `requests * 1.5`), but I can leave it unchanged.